release-notes/swan-lake-beta2-release-note.md

```diff
@@ -52,10 +52,45 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Beta2](https://github
 
 ### Standard Library Updates
 
+The new `postgresql` database package is introduced with this release. This package provides the functionality required to access and manipulate data stored in a PostgreSQL database.
+
 #### New Features
 
+##### Log Package
+Added Observability span context values to log messages when observability is enabled.
+
+##### I/O Package
+Introduced the `io:fprint` and `io:fprintln` APIs.
+```ballerina
+io:fprint(io:stderr, "Unexpected error occurred");
+io:fprintln(io:stderr, "Unexpected error occurred");
+io:fprint(io:stdout, "Passed without an error");
+io:fprintln(io:stdout, "Passed without an error");
+```
+
 #### Improvements
 
+##### Database Packages
+Return type of the stream is now inferred as a second parameter to the query remote method.
```

Comment:
@praneesha @lahirusamith I have changed the database package note. Please review

Reply:
LGTM!